src/components/OfferCard/components/InclusionsGroup/InclusionsGroup.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, FeatureListItem, NakedButton, Text } from 'components';
+
+const InclusionGroup = ({
+  inclusionsLabel,
+  inclusionsCallToActionLabel,
+  onClickInclusionsCallToAction,
+  ...inclusionsProps
+}) => {
+  return (
+    <>
+      <Box mb={3}>
+        <Text fontSize="base">{inclusionsLabel}</Text>
+      </Box>
+
+      {inclusionsProps.inclusions.map((item, key) => (
+        <FeatureListItem key={key} {...item} />
+      ))}
+
+      {inclusionsCallToActionLabel && (
+        <Box fontSize="sm" mt={3}>
+          <NakedButton
+            textDecoration="underline"
+            onClick={onClickInclusionsCallToAction}
+          >
+            {inclusionsCallToActionLabel}
+          </NakedButton>
+        </Box>
+      )}
+    </>
+  );
+};
+
+InclusionGroup.propTypes = {
+  inclusions: PropTypes.arrayOf(PropTypes.shape(FeatureListItem.propTypes)),
+  inclusionsLabel: PropTypes.string.isRequired,
+  inclusionsCallToActionLabel: PropTypes.string,
+  onClickInclusionsCallToAction: PropTypes.func,
```

Comment:
```suggestion
  label: PropTypes.string.isRequired,
  callToAction: PropTypes.shape({
    label: PropTypes.string,
    onClick: PropTypes.func,
  }),
```
potential props naming change? Given the component is called `InclusionsGroup`, its known the props are related to inclusions